### PR TITLE
Created example package

### PR DESCRIPTION
**Pull request by LeonardoMolleker**
ezgif com-gif-maker (10)

Created example in the project and corrected route of the package

### CODE REVIEW

File: lib/src/widgets/home_list_view.dart [hunk 1]
```diff
@@ -1,53 +1,118 @@
...
+  @override
+  _HomeListViewState createState() => _HomeListViewState();
+}
+
+class _HomeListViewState extends State<HomeListView> {
+  late List<RouteElement> _listRoute;
+
+  @override
+  void initState() {
+    // TODO: implement initState
```
Comment (MarcosCGlobant) [inline lib/src/widgets/home_list_view.dart]: you missed a comment here

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

File: lib/src/widgets/home_list_view.dart (hunk 1 above)
Comment (LeonardoMolleker) [reply lib/src/widgets/home_list_view.dart]: removed

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**